src/ophyd_async/epics/_backend/_aioca.py

```diff
@@ -49,7 +50,10 @@ def write_value(self, value) -> Any:
         return value
 
     def value(self, value: AugmentedValue):
-        return value
+        # for channel access ca_xxx classes, this
+        # invokes __pos__ operator to return an instance of
+        # the builtin base class
+        return +value
```

Comment:
This does a copy, which is fine for the scalars, but please can we make `CaArrayConverter.value` do `np.array(value, copy=False)`?
...